packages/css/native/shared/Css_Js_Core.ml

```diff
@@ -1064,7 +1142,12 @@ let transformOrigin3d x y z =
       ^ Length.toString z
       ^ {js| |js} )
 
-let transformBox x = D ({js|transform-box|js}, TransformBox.toString x)
+let transformBox x =
+  D
+    ( {js|transform-box|js},
```

Comment:
transformBox

Reply:
Done in https://github.com/davesnx/styled-ppx/pull/396/commits/35494bf086fd0b96418ad6f8d172edd2f5f1350b. Thanks!